website/core/Footer.js

```diff
@@ -21,10 +21,6 @@ const GithubButton = props => (
   </a>
 );
 
-GithubButton.propTypes = {
-  config: React.PropTypes.object
-};
-
```

Comment:
Only place we use it. It was logging an error each build. It isn't worth importing the 'proper' way unless we plan to use proptypes in more places than this IMHO.